setup.py

```diff
@@ -19,31 +19,45 @@ def _post():
     nltk.download('stopwords')
     nltk.download('punkt')
 
+
 class PostInstallCommand(install):
     """Post-installation for installation mode."""
     def run(self):
         install.run(self)
         _post()
 
+
 class PostDevelopCommand(develop):
     """Post-installation for development mode."""
     def run(self):
         develop.run(self)
         _post()
 
 
+# pypi does not render markdown.
+# https://stackoverflow.com/a/26737672/19741
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except(IOError, ImportError):
+    long_description = open('README.md').read()
```

Comment:
NICE!! Though, this is going to result in an odd-looking landing page for pypi package, I would suggest that if pypandoc is not installed, that `long_description` be something generic that describes the package (good enough for local installations, development, etc), and that the user is warned about pandoc not being available and the consequences of this.